README.md

```diff
@@ -2,10 +2,12 @@
 
 [![Build Status](https://circleci.com/gh/mapbox/mapbox-gl-rtl-text.svg?style=shield)](https://circleci.com/gh/mapbox/mapbox-gl-rtl-text)
 
-An [Emscripten](https://github.com/kripken/emscripten) port of a subset of the functionality of [International Components for Unicode (ICU)](http://site.icu-project.org/) necessary for [Mapbox GL JS](https://github.com/mapbox/mapbox-gl-js) to support [right to left text rendering](https://github.com/mapbox/mapbox-gl/issues/4). Support languages that use the Arabic script along with scripts that render right-to-left but don't require special character shaping (e.g. Hebrew).
+An [Emscripten](https://github.com/kripken/emscripten) port of a subset of the functionality of [International Components for Unicode (ICU)](http://site.icu-project.org/) necessary for [Mapbox GL JS](https://github.com/mapbox/mapbox-gl-js) to support [right to left text rendering](https://github.com/mapbox/mapbox-gl/issues/4). Supports the Arabic and Hebrew languages, which are written right-to-left. Mapbox Studio loads this plugin by default.
 
 **Requires [mapbox-gl-js](https://github.com/mapbox/mapbox-gl-js) (version 0.32.1 and up).**
 
+The [`mapbox-gl-language`](https://github.com/mapbox/mapbox-gl-language/) plugin, which automatically localizes labels into Arabic, is another option for supporting right-to-left text.
```

Comment:
The gl-language plugin displays Arabic name data (among other languages), while this plugin adds support for displaying Arabic names. A page requiring Arabic names would at a minimum install the gl-rtl-text plugin; to display the actual names, the page could use a specially modified style, manipulate the style at runtime, or just install the gl-language plugin for convenience.

Reply:
thanks for clarifying! will update.